src/sim-host/ui/theme.js

```diff
@@ -85,5 +85,36 @@ module.exports = {
         }
     },
 
-    defaultFontSize: 13
+    defaultFontSize: 13,
+
+    doCustom: function (css, themeObject) {
+        // Support scroll bar colors. Note that this is only supported for IE, because Webkit/Chrome makes it too difficult.
+        var scrollBarColors = themeObject.scrollbar && themeObject.scrollbar[''];
+        if (scrollBarColors) {
+            var ieScrollBarColorCssProperties = [];
+            if (scrollBarColors.background) {
```

Comment:
What is this check for? It seems to be duplicated further down, and not used elsewhere.